languages/cpp/exercises/concept/enums/strings_test.cpp

```diff
@@ -0,0 +1,87 @@
+#include "strings.h"
```

Comment:
This file `languages/cpp/exercises/concept/enums/strings_test.cpp` should be named `languages/cpp/exercises/concept/enums/enums_test.cpp`. This also holds for similarly named files in this PR.